www/scipylib/building/windows.rst

```diff
@@ -5,136 +5,131 @@ Building From Source on Windows
+on all of these projects, the Mingwpy_ website is an **excellent** source of more in-depth
+information than this document will provide.
+
+.. _Mingwpy: http://mingwpy.github.io/
+.. _ATLAS: http://math-atlas.sourceforge.net/
+.. _OpenBLAS: https://github.com/xianyi/OpenBLAS
+.. _LAPACK: http://www.netlib.org/lapack/faq.html
 
-Downloading and installing a compiler
--------------------------------------
+Compilers
+---------
 
-Generally you only need one C compiler (and one Fortran
-compiler). Cygwin is required if you want to build ATLAS
-yourself. Unless you are building from released source packages, the
-`Cython <http://cython.org/>`__ compiler is also needed.
+In order to build NumPy and SciPy, two compilers are needed: a C compiler
+and a Fortran compiler. The latter is technically not necessary for NumPy,
+but it is **strongly encouraged** to have one in order to build libraries like
+LAPACK_ or ATLAS_ that will significantly improve performance. For the remainder
+of this document, given the performance differences, **NumPy will be treated as if
+it actually does require such libraries, hence necessitating a Fortran compiler.**
 
 MinGW
 #####
 
-The easiest way to install MinGW is to use the MinGW installer from here. The
-current (as of 3 August, 06) candidate distribution with gcc and g77 3.4.5 is
-reported to work best. After installation you need to put the MinGW\bin
-directory on the path.
+The MinGW_ project provides Windows versions of the free GNU compilers gcc and
+gfortran. These are the compilers most NumPy SciPy developers work with and hence
+the compilers that are supported the best by build scripts in both libraries. Also,
+as indicated in the name, they also form the basis of the ongoing Mingwpy_ project
+described previously. Thus, from a long-term perspective, these compilers may be
+the optimal ones to use. Installation instructions can be found `here <http://www.mingw.org/wiki/InstallationHOWTOforMinGW>`__.
 
 Cygwin
 ######
 
-Cygwin can be conveniently installed and updated with their 
-`convenient installer <http://www.cygwin.com/setup.exe>`.
+A POSIX compatible Linux-like environment for Windows, Cygwin_ is a very useful tool,
+as it allows to compile and use many Unix tools without modification. It can also be
+used to build libraries ATLAS_, which at the moment is very Unix-oriented, although
+that may be subject to change as we will discuss later on. If selected during installation,
+Cygwin_ also makes available its own versions of the MinGW_ compilers (by the command
+line option "-mno-cygwin" to gcc), which produce identical code. **There is no need to
+install the separate MinGW distribution when Cygwin is already installed.** Installation
+instructions for Cygwin_ can be found `here <https://cygwin.com/install.html>`__. When
+using the installer, **make sure to search for and select** packages with the keyword
+**gcc** or **mingw** in them. The installer may also miss several important DLL's as
```

Comment:
depends what @matthew-brett actually used, but plain "mingw" is unmaintained and obsolete, you should point to mingw-w64. Julia's README.windows.md has an up to date cygwin cross section that lists what we use.